### PR TITLE
Trigger monolithic container builds in the core repo

### PR DESCRIPTION
**Pull request by bdunne**
For: https://github.com/ManageIQ/manageiq/pull/22192

### CODE REVIEW

File: .github/workflows/build_pods.yaml [hunk 1]
```diff
@@ -17,3 +17,18 @@ jobs:
...
       run: bin/build -t latest -n -p
+  notify_builders:
+    needs: build_pods
+    if: github.repository_owner == 'ManageIQ'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Notify manageiq-pods on RPM build
+      uses: peter-evans/repository-dispatch@v2
+      with:
+        token: ${{ secrets.DEPLOY_TOKEN }}
```
Comment (Fryguy) [inline .github/workflows/build_pods.yaml:28]: I'm fine with this for now, but the DEPLOY_TOKEN is currently be used for the automated go dependency thing - not sure if I want to create a second token or not.

File: .github/workflows/build_pods.yaml [hunk 2]
```diff
@@ -17,3 +17,18 @@ jobs:
       run: echo ${{ secrets.REGISTRY_PASSWORD }} | docker login docker.io --password-stdin --username ${{ secrets.REGISTRY_USERNAME }}
     - name: Build pods containers
       run: bin/build -t latest -n -p
+  notify_builders:
+    needs: build_pods
+    if: github.repository_owner == 'ManageIQ'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Notify manageiq-pods on RPM build
```
Comment (Fryguy) [inline .github/workflows/build_pods.yaml:25]:
```suggestion
    - name: Notify manageiq on RPM build
```

**Review by Fryguy — APPROVED** [review]
Minor comment.

File: .github/workflows/build_pods.yaml [hunk 3]
```diff
@@ -17,3 +17,18 @@ jobs:
       run: echo ${{ secrets.REGISTRY_PASSWORD }} | docker login docker.io --password-stdin --username ${{ secrets.REGISTRY_USERNAME }}
     - name: Build pods containers
       run: bin/build -t latest -n -p
+  notify_builders:
+    needs: build_pods
+    if: github.repository_owner == 'ManageIQ'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Notify manageiq on RPM build
```
Comment (bdunne) [inline .github/workflows/build_pods.yaml:25]:
```suggestion
    - name: Notify manageiq on pods build
```

**Pull request merged**